uportal-war/src/test/java/org/jasig/portal/groups/GroupsTester.java

```diff
@@ -449,15 +446,9 @@ public void testAddToALargeGroup() throws Exception
     assertNotNull(msg, dupBigGroup);
     msg = "Getting members from duplicate group.";
     print(msg);
-    Iterator itr = dupBigGroup.getChildren().iterator();
     msg = "Done retrieving members.  Will now count members.";
     print(msg);
-    int numMembers = 0;
-    while ( itr.hasNext() )
-    {
-        itr.next();
-        numMembers++;
-    }
+    int numMembers = dupBigGroup.getChildren().size();
```

Comment:
That's 2 times (so far).  This is a great example of why the old method signature was a problem.